pkg/kubelet/dockershim/docker_sandbox_windows.go

```diff
@@ -0,0 +1,39 @@
+// +build windows,!dockerless
+
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dockershim
+
+import (
+	dockertypes "github.com/docker/docker/api/types"
+	dockercontainer "github.com/docker/docker/api/types/container"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// applySandboxOptions applies LinuxPodSandboxConfig to dockercontainer.HostConfig and dockercontainer.ContainerCreateConfig.
+func (ds *dockerService) applySandboxOptions(hc *dockercontainer.HostConfig, c *runtimeapi.PodSandboxConfig, createConfig *dockertypes.ContainerCreateConfig, image string, separator rune) error {
+	wc := c.GetWindows()
```

Comment:
Is there a corresponding PR over in https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.pb.go to introduce a GetWindows() for runtimeapi.PodSandboxConfig? Note that it only has GetLinux() today.

Reply:
I added the `GetWindows()` method to the current pr's `staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto` file.

Reply:
Should coordinate changes to Containerd to support this additional CRI field as well.